jenkins/plugins.txt

```diff
@@ -0,0 +1,85 @@
+bouncycastle-api:2.16.1
```

Comment:
Most of these plugins require updates. But, I guess we are covering this as part of a separate work item.

Reply:
Yes. We will handle it in jenkins security related story.